### PR TITLE
fix defination of sc for streaming

### PR DESCRIPTION
**Pull request by NeoZhangJianyu**

### CODE REVIEW

File: notebooks/spark_basics/structured_streaming.ipynb [hunk 1]
```diff
@@ -47,7 +47,7 @@
     "from pyspark import SparkContext\n",
     "from bigdl.util.common import *\n",
     "\n",
-    "sc = SparkContext('local')\n"
+    "sc = SparkContext('local[2]')\n"
```
Comment (wzhongyuan) [inline notebooks/spark_basics/structured_streaming.ipynb]: why 2 ?

Comment (NeoZhangJianyu) [reply notebooks/spark_basics/structured_streaming.ipynb]: It's the min number of multiple threads.
how do you think local[*] ?
It means to use all of logic cores.

Comment (wzhongyuan) [reply notebooks/spark_basics/structured_streaming.ipynb]: 2 is a tricky number, if it's not specifically required to be 2,  * may be better

Comment (NeoZhangJianyu) [reply notebooks/spark_basics/structured_streaming.ipynb]: OK, I will update it to [*] soon.

**Pull request merged**